ext/oauth-client/Civi/Api4/Action/OAuthSysToken/Refresh.php

```diff
@@ -38,10 +38,15 @@ class Refresh extends BasicBatchAction {
 
   private $syncFields = ['access_token', 'refresh_token', 'expires', 'token_type'];
   private $writeFields = ['access_token', 'refresh_token', 'expires', 'token_type', 'raw'];
+  private $selectFields = ['id', 'client_id', 'access_token', 'refresh_token', 'expires', 'token_type', 'raw'];
```

Comment:
I agree this change is good and necessary to address the regression.

Just to note that the select-list is now smaller than it was before. 

```
+---------------------+
| name                |
+---------------------+
| id                  | yes
| client_id           | yes
| token_type          | yes
| access_token        | yes
| expires             | yes
| refresh_token       | yes
| raw                 | yes
| tag                 | no longer
| grant_type          | no longer
| scopes              | no longer
| resource_owner_name | no longer
| resource_owner      | no longer
| error               | no longer
| created_date        | no longer
| modified_date       | no longer
+---------------------+
```

("yes" - restored as part of the list; "no longer" - would've matched `*` but no longer matches the list)

AFAICS, this list does cover the most important fields for a typical refresh. I can't spot/think of any specific problems with the smaller list. Just noting the change in case there are edgey issues later or in case it triggers any thoughts.